poolrpc/auctioneer.proto

```diff
@@ -997,52 +996,6 @@ message TermsResponse {
     uint64 next_batch_clear_timestamp = 7;
 }
 
-message RelevantBatchRequest {
```

Comment:
I think it is okay removing this (which technically is a breaking change?) since it didn't work anyway.

Reply:
FWIW, this'll end up breaking a portion of the server once we update it eventually. However, we'll need to still keep this in place for older clients, so I think we need to keep it after all until we no longer see any queries for the call.